geppetto.js/geppetto-client/js/pages/admin/admin.ejs

```diff
@@ -3,14 +3,14 @@
 <head>
     <meta charset="utf-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1">
-    <link rel="icon" type="image/png" href="./geppetto/node_modules/@geppettoengine/geppetto-client/geppetto-client/style/favicon.png"/>
+    <link rel="icon" type="image/png" href="./geppetto/node_modules/@metacell/geppetto-meta-client/geppetto-client/style/favicon.png"/>
```

Comment:
not part of this issue, but because I see it: can we remove the pages completely?

And the paths are broken because of the assumed "./geppetto" folder and the additional `/geppetto-client`

Reply:
That was an automatic search-replacement that I missed. Agree we should remove the admin page. I'd like to do it and more pruning as a part of this refactoring because there are many things that may break and are useless considering the new workflows

Reply:
Removed the admin page